H5DensityWriter bug

This bug is in the form of a warning. It is due to a careless type conversion. This is an associated warning message:

./src/H5DensityWriter.hpp:140:85: warning: narrowing conversion of ‘numberOfVoxels[2]’ from ‘int’ to ‘hsize_t {aka long long unsigned int}’ inside { } is ill-formed in C++11 [-Wnarrowing]

TO DO: Solve the type conversion problem.

--- a/src/H5DensityWriter.hpp
+++ b/src/H5DensityWriter.hpp
@@ -137,9 +137,9 @@ class H5DensityWriter
 //      file->createGroup("/help")
       
       /* Create dataset densdataset */
-      hsize_t densdims[3] = {numberOfVoxels[0],/* Create dataspace ... */
-                             numberOfVoxels[1],
-                             numberOfVoxels[2]};
+      hsize_t densdims[3] = {hsize_t(numberOfVoxels[0]),/* Create dataspace ... */
+                             hsize_t(numberOfVoxels[1]),
+                             hsize_t(numberOfVoxels[2])};
       H5::DataSpace densspace(3, densdims);
       H5::DataSet * densdataset;/* Create dataset ... */
       densdataset = new H5::DataSet(file->createDataSet(

